sdk/storage/storage-datalake/package.json

```diff
@@ -1,5 +1,6 @@
 {
   "name": "@azure/storage-datalake",
+  "sdk-type": "client",
```

Comment:
This library is not a client library. In fact I think there are a couple that are track 1 vs 2. We should confirm with someone on the JS team to get the correct set.

Reply:
Removed.